Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volume-clone.md

```diff
@@ -17,8 +17,12 @@ for more info.
 
 In [pvc-clone](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pvc-clone.yaml),
 `dataSource` should be the name of the `PVC` which is already created by RBD
-CSI driver. The `dataSource` kind should be the `PersistentVolumeClaim` and also storageclass
-should be same as the source `PVC`.
+CSI driver. The `dataSource` kind should be the `PersistentVolumeClaim`.
+The `storageClassName` can be any storageclass (not necessarily same as Parent PVC) with the
+following conditions:
+
+* `provisinor` should be same for both the Parent PVC and the Clone PVC.
```

Comment:
does clone doesnt work cross operator? LIke in a DR operation where there maybe two rook operator?

Reply:
In the case of 2 different rook operators, the provisioner name will be different too right? hence it won't work for that case

Reply:
so we can say clone doesn't work cross-cluster....